ai_worker/src/analysis/article_840_tagger.py

```
@@ -0,0 +1,308 @@
+            )
+
+        # 가중치 기반 정렬 (weight가 높고, 매칭 키워드가 많은 순서)
+        sorted_categories = sorted(
+            category_matches.items(),
+            key=lambda x: (x[1]["weight"] * x[1]["count"], x[1]["count"]),
+            reverse=True
+        )
+
+        # 상위 카테고리들 선택 (GENERAL 제외)
+        selected_categories = []
+        for category, match_data in sorted_categories:
```

Ruff (0.14.5)

Loop control variable `match_data` not used within loop body

Rename unused `match_data` to `_match_data`

(B007)